app/core/settings.py

```diff
@@ -88,6 +91,47 @@
     'DATETIME_FORMAT': "%Y-%m-%dT%H:%M:%SZ",
 }
 
+LOGGING = {
+    'version': 1,
+    'disable_existing_loggers': False,
+    'formatters': {
+        'verbose': {
+            'format': "[%(asctime)s] %(levelname)s [%(name)s:%(lineno)s] %(message)s",
+            'datefmt': "%d/%b/%Y %H:%M:%S"
+        },
+        'simple': {
+            'format': '%(levelname)s %(message)s'
+        },
+    },
+    'handlers': {
+        'file': {
+            'level': 'DEBUG',
+            'class': 'logging.FileHandler',
+            'filename': os.path.join(DEFAULT_LOG_DIR, '%s.log' % PROJECT_NAME),
```

Comment:
hence cold be `%s.json.log`

Reply:
https://github.com/elifesciences/digests-formula/blob/master/salt/digests/config/etc-syslog-ng-conf.d-digests.conf#L4

Reply:
(that configuration for upload to Loggly ignores `uwsgi.log` but it's not much use there)